### PR TITLE
[DOCFIX] Improve Hive integration doc 

### PR DESCRIPTION
**Pull request by apc999**
- Clarify whether to set Alluxio as default fs
- Word tuning

This PR is motivated by [a mailing list question](https://groups.google.com/forum/#!topic/alluxio-users/ehkVIcyyfdA)

### CODE REVIEW

File: docs/en/Running-Hive-with-Alluxio.md [hunk 1]
```diff
@@ -35,23 +35,26 @@ export HIVE_AUX_JARS_PATH={{site.ALLUXIO_CLIENT_JAR_PATH}}:${HIVE_AUX_JARS_PATH}
 
 Alternatively, advanced users can choose to compile this client jar from the source code. Follow the instructs [here](Building-Alluxio-Master-Branch.html#compute-framework-support) and use the generated jar at `{{site.ALLUXIO_CLIENT_JAR_PATH_BUILD}}` for the rest of this guide.
 
-## Create Hive Tables on Alluxio
+## Use Alluxio as One Option to Store Hive Tables
```
Comment (gpang) [inline docs/en/Running-Hive-with-Alluxio.md]: Should this say "1. ...", and the other section say "2. ..."?

File: docs/en/Running-Hive-with-Alluxio.md [hunk 2]
```diff
@@ -173,7 +176,11 @@ location back to HDFS:
 hive> alter table TABLE_NAME set location "hdfs://namenode:port/table/path/in/HDFS";
 ```
 
-## Use Alluxio as Default Filesystem
+Instructions and examples till here illustrate how to use Alluxio as one of the filesystems to store
+tables in Hive, together with other filesystems like HDFS. THey doo not require to change the global
```
Comment (gpang) [inline docs/en/Running-Hive-with-Alluxio.md]: "THey" -> "They"
"doo" -> "do"

File: docs/en/Running-Hive-with-Alluxio.md (hunk 1 above)
Comment (apc999) [reply docs/en/Running-Hive-with-Alluxio.md]: thanks. that's what I want too. will fix

**Comment by apc999** [comment]
@gpang  PTAL

**Pull request merged**